cmds/ocm/commands/ocmcmds/common/utils.go

```diff
@@ -70,16 +71,19 @@ type OptionWithSessionCompleter interface {
 	CompleteWithSession(ctx clictx.OCM, session ocm.Session) error
 }
 
-func CompleteOptionsWithSession(ctx clictx.Context, session ocm.Session) options.OptionsProcessor {
+func CompleteOptionsWithSession(ctx clictx.Context, session ocm.Session, all ...bool) options.OptionsProcessor {
+	other := general.Optional(all...)
```

Comment:
This conveys no information for me. Why are we using this? What does this accomplish? What is `other`? Why is this needed here? What is `Complete` doing? What is `Configure` doing? This code doesn't explain anything and the names don't mean anything.

Please add some documentation or comments or rename some variables.

Reply:
see PR docu

Reply:
The PR docu is not in the code. If I look at this code a couple days from now, I won't be trying to look up the PR to find what's going on in there. Please add documentation directly on the function.